src/Ztm.Data.Entity.Postgres.Tests/TheoryWhenEnvIsSetAttribute.cs

```diff
@@ -0,0 +1,16 @@
+using System;
+using Xunit;
+
+namespace Ztm.Data.Entity.Postgres.Tests
+{
+    public class TheoryWhenEnvIsSetAttribute : TheoryAttribute
```

Comment:
IMO class name look weird. My idea is rename it to `ConditionalTheoryAttribute` with `RequiredEnv` property to specify the property name.

Reply:
Good idea